lib/raven/instance.rb

```diff
@@ -90,13 +98,17 @@ def send_event(event)
     #     MyApp.run
     #   end
     def capture(options = {})
```

Comment:
Ah, sorry, this was an artifact of the 2 step process. In the 1st step, I had to remove the `install_at_exit_hook` during the copy to get Instance to pass the specs, then made it match the initial implementation when Base delegated to it.